docs/api/events/js_kanban_setedit_event.md

```diff
@@ -13,14 +13,15 @@ description: You can learn about the set-edit event in the documentation of the
 ### Usage
 
 ~~~jsx {}
-"set-edit": ({ cardId: string | number } | null) => void;
+"set-edit": ({ cardId: string | number, eventSource: "select-card" } | null) => void;
 ~~~
 
 ### Parameters
 
 The callback of the **set-edit** event can take the *null* value or an object with the following parameter:
 
 - `cardId` - (required) the ID of the card to be edited
+- `eventSource` - (required) the ***"select-card"*** action that invokes the ***set-edit*** event
```

Comment:
no, not required. optional. see DMs from today's morning. it can be there if set-edit was called by select-card, or it can be not there if set-edit was called in some other way